tests/test_contracts.py

```diff
@@ -125,3 +125,24 @@ def test_scaffolder_extracts_events(scaffolder, test_filesystem):
     new_contract = scaffolder.from_abi(str(path), KNOWN_ADDRESS, "new_contract")
     new_contract.parse_events()
     assert new_contract.events
+
+
+def test_scaffolder_rejects_old_abi(scaffolder, test_filesystem):
+    """Test the scaffolder rejects pre-Solidity 0.6 ABIs."""
+    assert test_filesystem
+    path = Path() / "tests" / "data" / "old_abi.json"
+
+    with pytest.raises(ValueError) as exc_info:
```

Comment:
Maybe add a custom exeception, for unsupported SOlitidity language within execeptions